frontend/src/layout/navigation/Navigation.tsx

```diff
@@ -3,17 +3,18 @@ import { useValues } from 'kea'
 import { BillingAlerts } from 'lib/components/BillingAlerts'
 import React from 'react'
 import { sceneLogic } from 'scenes/sceneLogic'
+import { Scene } from 'scenes/sceneTypes'
 import { Breadcrumbs } from './Breadcrumbs/Breadcrumbs'
 import { DemoWarnings } from './DemoWarnings/DemoWarnings'
 import { SideBar } from './SideBar/SideBar'
 import { TopBar } from './TopBar/TopBar'
 
 export function Navigation({ children }: { children: any }): JSX.Element {
-    const { sceneConfig } = useValues(sceneLogic)
+    const { sceneConfig, activeScene } = useValues(sceneLogic)
 
     return (
         <Layout style={{ minHeight: '100vh' }}>
-            <TopBar />
+            {activeScene !== Scene.Ingestion && <TopBar />}
```

Comment:
Removing the top bar should probably be dependant on the same feature flag that adds the side bar. Otherwise, users in that side of the experiment can get trapped on the page.

Reply:
yes!! good catch, I totally missed that